src/Sarif/Autogenerated/SuppressionStates.cs

```diff
@@ -9,12 +9,12 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// <summary>
     /// The state of a result relative to a baseline of a previous run.
     /// </summary>
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.28.0.0")]
     [Flags]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.30.0.0")]
```

Comment:
`[Flags]` is now auto-generated by an argument to the `EnumHint`. (The attributes happen to come out in this order. I don't think it's worth controlling the order.)